### PR TITLE
#187364716 Integrate HoundCI with badge attached to readme

### PR DESCRIPTION
**Pull request by Impano1**
**What does this PR do?**
Integrate HoundCI with a badge attached to readme 

**Description of Task to be completed?**
- Validate the codebase with standard style guides

**How should this be manually tested?**
- Configurations are displayed on README.md file

**What are the relevant pivotal tracker/Trello stories?**
#187364716

### CODE REVIEW

**Comment by munezerobagira** [comment]
can you check your title and remember to add labels

**Review by mpanoathos — APPROVED** [review]

**Review by AngeloChristian1 — APPROVED** [review]

**Review by munezerobagira — APPROVED** [review]

**Review by munezerobagira — COMMENTED** [review]
Can you check reason why there is no houndci check on PR?

File: .hound.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#     enabled: true
+#     config_file: .eslintrc
+#     file_patterns:
+#       - '*.js'
+#       - '*.ts'
+#     enabled_plugins:
+#       - eslint-plugin-react
+#       - eslint-plugin-import
+#       - eslint-plugin-prettier
+#       - eslint-plugin:@typescript-eslint/recommended
```
Comment (munezerobagira) [inline .hound.yaml:17]:
```suggestion
```

**Review by munezerobagira — CHANGES_REQUESTED** [review]

File: .hound.yaml (hunk 1 above)
Comment (munezerobagira) [reply .hound.yaml:17]: is this comments or its part of configuration

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**